core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -775,6 +775,17 @@ class ZkUtils(val zkClient: ZkClient,
     consumersPerTopicMap
   }
 
+  def getTopicsPerMemberId(group: String, excludeInternalTopics: Boolean) : mutable.Map[String, List[String]] = {
```

Comment:
I guess it would. There is an existing method before this one (`getConsumersPerTopic`) that also takes `excludeInternalTopics` and assumes no default value for it. I wanted to make the new method consistent with that one.